tests/core/env/test.sh

```diff
@@ -57,7 +57,7 @@ rlJournalStart
         rlPhaseEnd
 
         rlPhaseStartTest "Empty environment file ($execute)"
-            rlRun -s "tmt run -r -e @empty.yaml 2>&1"
+            rlRun -s "tmt run -r -e @empty.yaml"
```

Comment:
@happz can we also test that the quiet flag now silences the `warning` as the user wanted for the proof?

Reply:
It does not yet work this way, yet. This PR points `print()` to `stdout`, there will be another patch changing how `--quiet` works. I added a test though, in 5e471f82, it's commented out.